### PR TITLE
[sosreport] Add gluster deployment and cleanup log

### PR DESCRIPTION
**Pull request by gobindadas**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [] Is the subject and message clear and concise?
- [] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

### CODE REVIEW

**Comment by gobindadas** [comment]
@sandrobonazzola Can you please review and merge?

File: sos/plugins/ovirt_hosted_engine.py [hunk 1]
```diff
@@ -48,6 +48,14 @@ def setup(self):
             '/var/log/ovirt-hosted-engine-ha/agent.log',
             '/var/log/ovirt-hosted-engine-ha/broker.log',
         ])
+
+        # Add gluster deployment and cleanup log
+        self.add_copy_spec([
+            '/var/log/cockpit/ovirt-dashboard',
```
Comment (pmoravec) [inline sos/plugins/ovirt_hosted_engine.py]: Collecting whole dir implies collecting the two particular logfiles from the dir - didnt you want to collect some different directory here?

Comment (gobindadas) [reply sos/plugins/ovirt_hosted_engine.py]: No, The logfiles which are mentioned under '/var/log/cockpit/ovirt-dashboard' need to be included in sosreport.

Comment (TurboTurtle) [reply sos/plugins/ovirt_hosted_engine.py]: Right, so you only need the first line collecting the dir. Everything in that directory will be collected, including those files.

Comment (gobindadas) [reply sos/plugins/ovirt_hosted_engine.py]: @pmoravec Ok got it. I have pushed the changes.

**Comment by sandrobonazzola** [comment]
> @sandrobonazzola Can you please review and merge?

I have no merge rights here. I can start telling you that you should edit the PR to confirm you adhere to contribution guidelines.

**Comment by sandrobonazzola** [comment]
@bmr-cymru can you please review?

**Comment by pmoravec** [comment]
Code is OK, just commit subject + message needs improvement (see Contrib. guidelines), like:

    [ovirt_hosted_engine] Add gluster deployment and cleanup log

    Resolves: #1748 

(since the future commit will resolve this upstream issue/PR, not some bugzilla.redhat.com BZ (yet) )

**Pull request closed without merging**